package.json

```diff
@@ -76,9 +76,9 @@
     "@types/npm-package-arg": "6.1.1",
     "@types/semver": "7.5.0",
     "@types/ungap__structured-clone": "0.3.0",
-    "@typescript-eslint_bottom/eslint-plugin": "npm:@typescript-eslint/eslint-plugin@5.52.0",
-    "@typescript-eslint_bottom/parser": "npm:@typescript-eslint/parser@5.52.0",
-    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@typescript-eslint_bottom/eslint-plugin": "npm:@typescript-eslint/eslint-plugin@6.0.0",
+    "@typescript-eslint_bottom/parser": "npm:@typescript-eslint/parser@6.0.0",
+    "@typescript-eslint/eslint-plugin": "6.1.0",
```

Comment:
The version for @typescript-eslint/eslint-plugin, @typescript-eslint/parser and has to be consistent: either 6.0.0 or 6.1.0.